### ISSUE TITLE
Question of the .json payload

### ISSUE THREAD

**Issue by hfanieng** [nzoschke/vizlink#26]
great project 👍 

{"payload":{"code":0,"msg":"hi"},"ms":1711834477545,"type":"sys","version":1}

is there a reason why the tuples "ms", "type" and "sys" are not included in the payload? i struggle a little bit by implement this in my python code.

**Comment by nzoschke** [comment]
The general idea is a "message" is an envelope for a "payload".

Every message has a "type" and that value lets you know how to further decode the payload.

**Comment by nzoschke** [comment]
This reminds me I have some typescript and Go types and examples of decoding messages in one of my consumer programs that I can get in here to help document.

Also it's still very early in the project so I'm open for feedback and changing the messages to make them easy to consume.

**Comment by hfanieng** [comment]
> This reminds me I have some typescript and Go types and examples of decoding messages in one of my consumer programs that I can get in here to help document.
> 
> Also it's still very early in the project so I'm open for feedback and changing the messages to make them easy to consume.

now problem - I'm also very early in my projects 😉
very great idea from you to this with BLT 👍 
have to raise my knowledge with .json & python - now i can parse everything that viz link sends 💯

**Comment by nzoschke** [comment]
That's great news. Are the pre-compiled releases working? What OS and architecture are you using?

**Comment by hfanieng** [comment]
I'm on MacOS primary, have also Windows as backup.

actually for all testing environment the XDJ-XZ is preferred. also got RB-DMX-1, the XP-2 and DDJ-800 + DDJ-400.

**Comment by nzoschke** [comment]
Nice, Mac OS and CDJ-3000s here. I've done a little bit of testing with friends XZ and Windows but nothing major.

I would love to figure out how we can get the equivalent of Pro Link / beat-link beats and phrases out of the DDJ controllers and Rekordbox performance mode someday...

**Comment by hfanieng** [comment]
I don't think that AlphaTheta will give us the option for the DDJ (have the 400 and 800) and the rekordbox 😞
